src/app/screens/Discover/websites.json

```diff
@@ -153,6 +153,12 @@
         "subtitle": "The blue bird experience for Nostr",
         "logo": "https://cdn.getalby-assets.com/alby-extension-website-screen/yosup_thumbnail.svg",
         "url": "https://yosup.app/"
+      },
+      {
+        "title": "Primal",
+        "subtitle": "Nostr highlights by Primal",
+        "logo": "https://primal.net/assets/logo_fire-409917ad.svg",
```

Comment:
You can use this one now:
https://cdn.getalby-assets.com/alby-extension-website-screen/primat-logo_fire-409917ad.svg

Reply:
Okay, I will make changes